File: P_2/Examen_P2/Napakalaki_Ruby_P2/lib/prize.rb
#encoding: utf-8
# MADE BY: JOSE MIGUEL HDEZ GARCIA & JAIRO ABRIL MOYA

class Prize
    attr_accessor :treasures
    attr_accessor :level
 
    # Metodo initialize que inicia los atributos
    def initialize(treasures = 0, level = 0)
        # Atributos
        @treasures = treasures
        @level = level
    end

    # Metodo to_string para pintar en pantalla lo deseado
    def to_s
        "Tesoros ganados: #{@treasures.to_s}
        Niveles ganados: #{@level.to_s}"
    end
end

File: P_5/Napakalaki_Java_P5/src/napakalaki/Prize.java
// MADE BY: JOSE MIGUEL HDEZ GARCIA & JAIRO ABRIL MOYA

package napakalaki;

/*
* Clase Prize del paquete napakalaki
*/

public class Prize{
    
    // Declaramos los atributos PRIVADOS a utilizar
    private int level;
    private int treasures;

    // Constructor
    public Prize(int treasure, int level){
        
        treasures = treasure;
        level = level;
    }

    // Consultores básicos
    // Devuelve los tesoros que se pueden ganar con el buen rollo
    public int getTreasures(){
        return treasures;
    }

    // Devuelve los niveles que pueden ganarse con el buen rollo
    public int getLevel(){
        return level;
    }

    // Método toString para pintar una secuencia
    public String toString(){
        return "Level -> " + Integer.toString(level) + 
               ", Treasures -> " + Integer.toString(treasures);
    }
}